package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ffc-pay-statement-constructor",
-  "version": "0.7.0",
+  "version": "0.7.1",
```

Comment:
This will conflict with pull/11 after merge. Also why 0.7.1 but not 0.8.0 ?

Reply:
Based version off `main` as we dnt _know_ (altho like 99% sure!) which PR will merge 1st, could rebase if needs be

Did patch as functionality isn't called anywhere yet

Happy to bump as minor and upto `0.8.0` to avoid merge conflicts